### ISSUE TITLE
Extending compatibility of Molecule class instantiation from other file types

### ISSUE THREAD

**Issue by ianrinehart** [SEDenmarkLab/molli_firstgen#11]
As a normal part of doing any computations, a bunch of structures that I was working with had some errors. I could easily get xyz files, triage the problems, and make the necessary edits so that they will work. 

Unfortunately, I have xyz files and there is not a clear way to create molecule class instances. If there were, then I would easily be able to reenter the workflow with these "special" cases. 

In other words, can there be a ml.Molecule.from_xyz() and a condition for the .from_file() function to accomodate xyz?
